### DISCUSSION TITLE
table filter

### DISCUSSION THREAD

**Discussion by Pshemas** [jgm/pandoc#9975] [Q&A]
Are there any filters / options to change the way tables are rendered when converting from markdown to pdf?

I'm not a huge fan of this style that's the default option in ... default tex and also eisvogel:

![image](https://github.com/user-attachments/assets/60056723-e667-4d2a-a01f-54895e0acb14)

I wonder whether there's some option to get tables with cell borders (horizontal / vertical lines), possibly coloring of the first (title) row, or alternating row colors - without actually writing latex.

I've spent past hour searching, but obviously I can't figure the magic keywords that would get me anything .

**Comment by Wandmalfarbe** [comment]
Eisvogel has the option `table-use-row-colors` that draws striped tables. Unfortunately that doesn't look good at the moment (see https://github.com/jgm/pandoc/issues/3929)
